app/assets/html/home.html

```diff
@@ -1,2 +1,3 @@
 <!-- TODO Implement main homepage view here -->
 <h1>News</h1>
+Select a person to start debating.
```

Comment:
Why was this added? I think telling the user to "select a person" is a little confusing because the user may not have matched with anyone yet (and therefore need to go though the matching process). Also, I don't think "debating" is quite the right vernacular we want to be using.